types/jss/index.d.ts

```diff
@@ -137,6 +138,10 @@ export declare class JSS {
  * Creates a new instance of JSS.
  */
 export function create(options?: Partial<JSSOptions>): JSS;
+/**
+ * Creates a GenerateClassName function
+ */
```

Comment:
that comment repeats the function below without adding any value :)

Reply:
@kof I did feel bad about that comment but 90% of the definitions have these kind of comments so I felt obligated to add it in the initial draft just for the sake of consistency.